dali/kernels/signal/moving_mean_square_args.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_SIGNAL_MOVING_MEAN_SQUARE_ARGS_H_
+#define DALI_KERNELS_SIGNAL_MOVING_MEAN_SQUARE_ARGS_H_
+
+namespace dali {
+namespace kernels {
+namespace signal {
+
+struct MovingMeanSquareArgs {
```

Comment:
Why not just pass these two integers to Setup and Run?

Reply:
I've seen that that's the general approach in other kernels. Honestly, I don't mind neither